modlauncher/src/main/resources/META-INF/services/cpw.mods.modlauncher.serviceapi.ILaunchPluginService

```diff
@@ -0,0 +1 @@
+net.neoforged.accesstransformer.ml.AccessTransformerService
```

Comment:
I find it weird that there is both `module-info.java` and this file. Surely this file should not be needed anymore? SJH will correctly process services declared in the `module-info.java` file.